src/Chapter2.hs

```diff
@@ -40,6 +40,8 @@ Now, if you are ready, bring it on!
 
 module Chapter2 where
 
+import Data.List ( foldl' )
```

Comment:
When I started reviewing this Chapter and saw this import, I already knew that something good is waiting! :smile: